### PR TITLE
Update publishing script

### PR DESCRIPTION
**Pull request by jscasca**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -61,9 +61,38 @@ node("primary") {
...
 
-  if (isReleaseBranch()) {
+  if (isReleaseBranch() && isReleasable()) {
     stage("Publish") {
       sh 'yarn run publish'
     }
   }
 }
+
+def isNewerVersion(String current, String next) {
```
Comment (lnewson) [inline Jenkinsfile:71]: Can these be added to `waluigi` instead? As you're going to need them for react and vue as well I believe?

Comment (jscasca) [reply Jenkinsfile:71]: `isNewerVersion` for sure. `isRelesable` depends on having the specific scripts in the integration so not sure that was a great idea. Thoughts?

Comment (lnewson) [reply Jenkinsfile:71]: if you update that to the following, then you shouldn't need the package.json scripts:
```
def isReleasable() {
  def packageName = sh(script: 'node -p -e "require(\'./package.json\').name"', returnStdout: true).trim()
  echo "Checking releasability for package: " + packageName
  if (packageName) {
    def remoteVersion = sh(script: "npm view ${packageName} version", returnStdout: true)
    echo "Validating remote version: " + remoteVersion
    def localVersion = sh(script: 'node -p -e "require(\'./package.json\').version"', returnStdout: true)
    echo "Validating local version: " + localVersion
    if (localVersion) {
      return isNewerVersion(remoteVersion, localVersion)
    }
  }
  return false
} 
```

Comment (lnewson) [reply Jenkinsfile:71]: For reference, `node -p -e "require('./package.json').name"` and `node -p -e "require('./package.json').version"` were the changes I made, as that should achieve the same thing.

Comment (lnewson) [reply Jenkinsfile:71]: I just read you can do `npm view . version` as well, which worked okay in my quick local testing.

File: package.json [hunk 2]
```diff
@@ -5,14 +5,16 @@
   "author": "Ephox Inc",
   "license": "Apache-2.0",
   "scripts": {
+    "package-name": "echo $npm_package_name",
```
Comment (lnewson) [inline package.json:8]: Do you still need these?

Comment (jscasca) [reply package.json:8]: Completely forgot about those. I mean they dont hurt anybody. But I will remove them nonetheless

**Pull request merged**